src/Nethermind/Nethermind.Serialization.SszGenerator/SszGenerator.cs

```diff
@@ -0,0 +1,448 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+using System.Text;
+using System.Text.RegularExpressions;
+
+[Generator]
+public partial class SszGenerator : IIncrementalGenerator
```

Comment:
Was this class generated or handcrafted? If the 2nd then t4 templates were more readable :/

Reply:
Not sure if t4 is even applicable here. Absolute most of the stuff is generated